scripts/build-image.sh

```diff
@@ -39,5 +39,11 @@ IMAGE_NAME=$(grep vm_name $TEMPLATE | awk '{print $2}' | sed -e 's/\"//g' | sed
 set -xe
 cd packer
 packer build -var "chef_version=$CHEF_VERSION" $(basename $TEMPLATE)
-qemu-img convert -o compat=0.10 -O qcow2 -c ${DIR_NAME}/${IMAGE_NAME}.qcow2 \
-  ${DIR_NAME}/${IMAGE_NAME}-compressed.qcow2
+
+if [ $(packer -v) == "0.7.5" ] ; then
```

Comment:
You need to change this to the following:

``` bash
if [ "$(packer version | grep ^Packer)" == "Packer v0.7.5" ] ; then
```

It needs to look at the EXACT string. Plus you need to put quotes around the command.